src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `books` (
+`id` BIGINT NOT NULL AUTO_INCREMENT,
+ `title` VARCHAR(250),
+ `price` DECIMAL,
+ PRIMARY KEY (`id`));
```

Comment:
```suggestion
 PRIMARY KEY (`id`));
 
 ```
Probobly need empty line